src/CheckoutSdk/SdkAuthorization.cs

```diff
@@ -15,16 +13,16 @@ public SdkAuthorization(PlatformType platformType, string credential)
             _credential = credential;
         }
 
-        public AuthenticationHeaderValue GetAuthorizationHeader()
```

Comment:
What is the purpose of removing this?

Reply:
We had to change AuthenticationHeaderValue  to string because the AuthenticationHeaderValue  constructor validates well formed token. Doing with this new code we can use TryAddWithoutValidation and avoid issues.